apps/aetestframework/test/multi_node_sync_test.exs

```diff
@@ -0,0 +1,147 @@
+defmodule MultiNodeSyncTest do
+  use ExUnit.Case
+
+  alias Aetestframework.Worker, as: TestFramework
+  alias Aecore.Chain.Worker, as: Chain
+  alias Aecore.Tx.Pool.Worker, as: Pool
+
+  setup do
+    TestFramework.start_link(%{})
+    Chain.clear_state()
```

Comment:
You should use new way of cleaning blockchain state, but I don't think you use the blockchain state at all so you might not need this.

Reply:
> You should use new way of cleaning blockchain state, but I don't think you use the blockchain state at all so you might not need this.

This comment still applies

Reply:
> You should use new way of cleaning blockchain state, but I don't think you use the blockchain state at all so you might not need this.

This comment still applies